app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -0,0 +1,241 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.core.widget.NestedScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/background"
+    tools:context=".ui.activities.MovieDetailsActivity">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:descendantFocusability="blocksDescendants">
+
+        <FrameLayout
+            android:id="@+id/imageView"
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
+            android:foreground="@drawable/mask"
+            app:layout_constraintDimensionRatio="375:298"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent">
+
+            <!-- orig -->
+            <ImageView
+                android:id="@+id/orig"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:contentDescription="@string/original_image" />
+        </FrameLayout>
+
+        <!-- Rectangle -->
+        <View
+            android:id="@+id/rectangle"
+            android:layout_width="@dimen/amd_rectangle_width"
+            android:layout_height="@dimen/amd_rectangle_height"
+            android:layout_marginStart="@dimen/spacing_normal_16"
+            android:layout_marginBottom="@dimen/spacing_small_8"
+            android:alpha="0.9"
+            android:background="@drawable/rectangle"
+            app:layout_constraintBottom_toTopOf="@+id/name"
+            app:layout_constraintStart_toStartOf="parent" />
+
+        <!-- 13+ -->
+        <TextView
+            android:id="@+id/pg"
+            android:layout_width="0dp"
+            android:layout_height="@dimen/spacing_normal_16"
+            android:layout_marginTop="@dimen/spacing_extra_small_4"
+            android:layout_marginBottom="@dimen/spacing_extra_small_4"
+            android:text="@string/pg"
+            android:textAppearance="@style/PG"
+            app:layout_constraintBottom_toBottomOf="@+id/rectangle"
+            app:layout_constraintEnd_toEndOf="@+id/rectangle"
+            app:layout_constraintStart_toStartOf="@+id/rectangle"
+            app:layout_constraintTop_toTopOf="@+id/rectangle" />
+
+        <!-- Name -->
```

Comment:
Are these comments really useful? I mean IDs basically contain the same name and are pretty readable. Can we avoid it?

Reply:
Deleted.